source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -263,9 +263,10 @@ def _getColumnOrderArrayRawOutProc(self, columnCount: int) -> Optional[ctypes.Ar
 				columnCount,
 				internalCoa
 			)
-			if res:

```

Comment:
Note that this appears to have been an earlier bug. I'm concerned about any implications of `readProcessMemory` not occurring correctly before this patch.

Reply:
What is the expected value of res? Better check against that, e.g. `if res == 0:` instead of its boolean value

Reply:
What is the earlier bug?

Reply:
This was a misunderstanding on my behalf